appveyor.yml

```diff
@@ -0,0 +1,91 @@
+version: 1.0-{build}
+image:
+- Visual Studio 2015
+configuration:
+- Debug
+- Release
+clone_depth: 1
+environment:
+  QT_VER: 5.11
```

Comment:
Why not 5.12 (LTS)?

Reply:
5.12 is not available for VS2015 in a 32-bit version: https://www.appveyor.com/docs/windows-images-software/#qt

Also, I wasn't aware of the LTS versioning for Qt.